### PR TITLE
update existing terminology

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

**Review by fluffy — COMMENTED** [review]
LGTM

**Review by suhasHere — DISMISSED** [review]
I might be voting to not update terminology section for -05

**Review by afrind — COMMENTED** [review]
These updates to the terminology seem to be editorial only and improving clarity.

LGTM

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -130,31 +130,32 @@ Commonly used terms in this document are described below.
...
+: The party initiating a transport session.
 
 Congestion:
 
 : Packet loss and queuing caused by degraded or overloaded networks.
 
 Consumer:
 
-: A QUIC endpoint receiving media over the network. This could be a consumer or a relay.
+: A QUIC endpoint receiving media over the network. This could be an endpoint or a relay.
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:141]: Wrong definition of endpoint

**Review by kixelated — CHANGES_REQUESTED** [review]

**Review by kixelated — APPROVED** [review]

**Pull request merged**